src/Etg.Yams.Core/YamsDiModule.cs

```diff
@@ -20,11 +20,13 @@ public class YamsDiModule
     {
         private readonly IContainer _container;
 
-        public YamsDiModule(YamsConfig config, IDeploymentRepository deploymentRepository,
+        public YamsDiModule(YamsConfig config,
```

Comment:
I'd keep the old constructor too to stay backward compatible

Reply:
Done